Apromore-Extras/APMLogModule/src/main/java/org/apromore/apmlog/LaLog.java

```diff
@@ -36,10 +35,7 @@
 import org.eclipse.collections.impl.map.mutable.UnifiedMap;
 import org.eclipse.collections.impl.set.mutable.UnifiedSet;
 
-import java.util.ArrayList;
-import java.util.BitSet;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
One of our Checkstyle rules is "don't use wildcarded imports".  It's not enforced by the build process...yet.  It would be better to do full import lists so there's less work to enable Checkstyle in future.